src/viewDirective.js

```diff
@@ -209,29 +237,6 @@ function $ViewDirective(   $state,   $injector,   $uiViewScroll,   $interpolate,
 
         updateView(true);
 
-        function cleanupLastView() {
-          if (previousEl) {
-            previousEl.remove();
```

Comment:
This code is important, and I don't see it in the new impl.  It ensures that the prior ui-view copies get properly destroyed.

This code allows views to be swapped in multiple times, even while the first instance is still animating out.  This can be seen by creating a very long "out" animation, then cycling back and forth between sibling states.